gatsby-node.js

```diff
@@ -6,10 +6,10 @@ exports.onCreateBabelConfig = ({ actions }, pluginOptions) => {
   });
 };
 
-exports.onCreateWebpackConfig = ({ actions, stage }) => {
+exports.onCreateWebpackConfig = ({ actions, stage }, {plugins, ...options}) => {
```

Comment:
Does this need the `plugins` destructured out?  Could it be:

`exports.onCreateWebpackConfig = ({ actions, stage }, pluginOptinos) => {`